src/System.IO.FileSystem/tests/Directory/Delete_MountVolume.cs

```diff
@@ -0,0 +1,461 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+/**
+This testcase attempts to delete some directories in a mounted volume
+ - Different drive is mounted on the current drive
+ - Current drive is mounted on a different drive
+ - Current drive is mounted on current directory
+ 	 - refer to the directory in a recursive manner in addition to the normal one
+**/
+using System;
+using System.IO;
+using System.Text;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+public class Directory_Delete_MountVolume
+{
+    private delegate void ExceptionCode();
+    private static bool s_pass = true;
+
+    [Fact]
+    [ActiveIssue(0)]
```

Comment:
Need something here other than just ActiveIssue(0).  Either file an issue or delete the test.